ui-tests/src/test/java/org/openmrs/reference/LoginTest.java

```diff
@@ -67,6 +71,7 @@ public void verifyNurseModulesAvailableOnHomePage() throws Exception {
     public void verifySysadminModulesAvailableOnHomePage() throws Exception {
```

Comment:
And here` also`

Reply:
True i aggree since adding a waitForPage handles that